_sidebar.md

```diff
@@ -1,11 +1,11 @@
 
 - [Welcome](/)
```

Comment:
Looks awesome, much better and really like the symbols.

The welcome link doesn't go anywhere, we could remove the link.